recipes/beaker-py/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set version = "1.6.7" %}
+
+package:
+  name: beaker-py
+  version: {{ version }}
+
+source:
+  url: https://github.com/allenai/beaker-py/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d1445637651e9a6ddbcce609108284a4d6b4ad3603e579a96dbdc1e77d9e0447
+
+build:
+  skip: true  # [win]
```

Doesn't this package qualify for `noarch: python`?

```suggestion
  noarch: python
```